README.Rmd

```diff
@@ -31,26 +31,26 @@ library(tiltPlot)
 ### 1. Sankey Plot 
 
 ```{r}
-sankey_toy_data
+financial_toy_data
```

Comment:
In the early stages of development it's likely we'll change the name of datasets frequently. We can reduce the impact by being a vague with:

```r
data <- financial_toy_data
```

Then the many calls to `plot_sankey(data)` won't change if the name of the dataset changes, e.g. from `financial_toy_data` to `financial` or whatever.

If the scope of the examples is large and you show multiple plots and multiple datasets, then `data` is too vague but still you can make the code a bit more robust to changes somehow, e.g.

```r
fin <- financial_toy_data
plot_sankey(fin)
```
 
This will be robust to many likely changes e.g. `financial_toy_data  > financial_data > finance > fin_data ...`

Reply:
Cool ! Thank you @maurolepore !